packages/nextjs-mf/src/plugins/NextFederationPlugin/apply-server-plugins.ts

```diff
@@ -41,9 +43,11 @@ export function applyServerPlugins(
 }
```

Comment:
The function 'applyServerPlugins' could be modularized further. The logic for each plugin could be extracted into separate functions for better maintainability and testing.